src/rgw/rgw_bucket.cc

```diff
@@ -1420,6 +1420,7 @@ static int bucket_stats(RGWRados *store, const std::string& tenant_name, std::st
 
   formatter->open_object_section("stats");
   formatter->dump_string("bucket", bucket.name);
+  formatter->dump_string("tenant", bucket.tenant);
```

Comment:
:+1:

Reply:
Hmmm, maybe, but this change is related to the issue fixed by this PR, so why not include it here?

Reply:
@votdev this PR is titled **mgr/dashboard**, so it's somewhat unexpected that it makes changes to a completely different Ceph component. 

I am aware this change is related, but as it touches RGW and introduces a change in behaviour (the result of an API call changes), I'd be in favor of giving it some more visibility :)

Reply:
Done, see https://github.com/ceph/ceph/pull/24895 and https://tracker.ceph.com/issues/36681

Reply:
Will not adapt the PR but rebase it after #24895 has been merged.